### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BenjaminGarza**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Great start to this project. But some few changes need to be made

File: main.css
Comment (Oghenebrume50) [inline main.css]: This line is affecting your code, the footer jumps up. You may want to look into heights and width, most times you don't need to explicitly set heights of a large container. and your width is off too, it leaves a big space that is scrollable, too big to be ignored.

File: index.html
Comment (Oghenebrume50) [inline index.html]: This article tag shouldn't be wrapping the nav tag.

File: index.html
Comment (Oghenebrume50) [inline index.html]: The content in this aside tag should be in the footer, and the images should be passed as background images, just like you did for the ones above. use a div and pass as background image

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Fix those things and submit for re-review, nice attempt, you did really good.

**Comment by Oghenebrume50** [comment]
Hi, it is best to use positioning for inner elements and not outer container currently this is how your project displays

**Tips**
Create an outer container to hold nav, main and footer Set a display: flex on it
Flex: direction column
avoid using margins on the nav, main, footer.
style the inner elements

File: main.css
Comment (Oghenebrume50) [inline main.css]: If you remove this margin you would see how it affects the page display. You have to be careful with margins

File: main.css
Comment (Oghenebrume50) [inline main.css]: You shouldn't set an explicit width like this on the body.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]

**Comment by samgaco** [comment]
Hi, @BenjaminGarza I can't see any changes after the last review. Is everything ok?. I wait for your reply.

**Comment by BenjaminGarza** [comment]
> Hi, @BenjaminGarza I can't see any changes after the last review. Is everything ok?. I wait for your reply.

Just made more changes today and putting in for another code review, thanks!

**Comment by bolah2009** [comment]
Currently under review

**Review by bolah2009 — APPROVED** [review]
## Nice Job [Ben](../../)  :clap: 

The [clone](https://raw.githack.com/BenjaminGarza/Backgrounds-and-Gradients/Development/index.html) looks good to me. :+1:

#### Consider checking these [errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FBenjaminGarza%2FBackgrounds-and-Gradients%2FDevelopment%2Findex.html) before merging.

#### Friendly reminder: You should use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in this project. However, changing it now would take too much time. Please make sure you use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in the next project. You will find instructions in the Pathwright lesson description.

Good luck with your next project.

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**
